### DISCUSSION TITLE
Error in Everything Search

### DISCUSSION THREAD

**Discussion by resherwni** [Flow-Launcher/Flow.Launcher#689] [Q&A] [answered]
Running the Portable version on Windows 10 64-bit. I installed the Everything plugin and when I run a search for a file I know exists I receive the error shown below:

Error while querying Everything
The Sort Option is not Fast Sort, it may take a long time to finish the query.

![Screenshot 2021-09-13 161052](https://user-images.githubusercontent.com/71411863/133032404-b88ba968-0ae3-4e22-b679-7db6216ddf28.png)

I have Everything installed and the same query word returns the expected result. What is the fix please?

**Comment by jjw24** [comment]
Hi there, I think it expects Everything service v1.4.1 or higher. Can u check the version of Everything you are using please. 

May also be an error related to this https://github.com/Flow-Launcher/Flow.Launcher.Plugin.Everything/issues/34

**Comment by resherwni** [comment]
I have Everything version Version 1.4.1.1009 Lite (x64). I tried exiting Everything as per the related error that you posted and it didn't change things. Other ideas?

**Comment by jjw24** [comment]
Could you reproduce the error then attach the log here please. Type `open log` to get to the directory and then attach the latest dated log please.

**Comment by jjw24** [comment]
Also need to just double check do you actually have Fast Sort option enabled for Everything:

![image](https://user-images.githubusercontent.com/26427004/133250024-d1f41bd6-3ce3-42bf-926e-8c6e3bb94454.png)

@taooceros please confirm if they are the fast sort options referred to in the screenshot

**Comment by resherwni** [comment, marked as answer]
Here are the Everything settings which yes, do include the Fast Sort options you have above

![Screenshot 2021-09-15 192106](https://user-images.githubusercontent.com/71411863/133407272-ab4c8304-727c-402f-8748-1d09838a378d.png)

Also, here is the log file

[2021-09-15.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/7168711/2021-09-15.txt)

**Reply by jjw24** [reply]
Your Everything is running right?

Can you also check what's set in the plugin settings and set the sort option to 'Name' if not:
![image](https://user-images.githubusercontent.com/26427004/133421312-2406ac30-c07f-4806-8fcb-3957389fd463.png)

**Comment by jjw24** [comment]
This should be addressed with the v1.5.4 Everything release ('pm update Everything') @resherwni.

For future issues please create an issue at  https://github.com/Flow-Launcher/Flow.Launcher.Plugin.Everything
